packages/dpkg/build.sh

```diff
@@ -72,13 +71,21 @@ share/polkit-1
 "
 
 termux_step_pre_configure() {
+	# `dpkg`'s configure script expects the `.dist-version` file to contain the package version.
+	# This is missing in the tarballs from salsa.debian.org/dpkg-team/dpkg
+	echo "$TERMUX_PKG_VERSION" > "$TERMUX_PKG_SRCDIR/.dist-version"
+
+	# `.dist-vcs-id` isn't strictly required, but configure complains if its missing.
+	git ls-remote https://salsa.debian.org/dpkg-team/dpkg.git \
+	| grep -F "refs/tags/$TERMUX_PKG_VERSION^{}" \
+	| cut -f1 > "$TERMUX_PKG_SRCDIR/.dist-vcs-id"
 	(
 		cd "${TERMUX_PKG_SRCDIR}" && \
 		./autogen && \
 		patch -p1 -i "${TERMUX_PKG_BUILDER_DIR}"/configure.diff
 	)
 	export TAR=tar # To make sure dpkg tries to use "tar" instead of e.g. "gnutar" (which happens when building on OS X)
-	perl -p -i -e "s/TERMUX_ARCH/$TERMUX_ARCH/" $TERMUX_PKG_SRCDIR/configure
+	perl -p -i -e "s/TERMUX_ARCH/$TERMUX_ARCH/" "$TERMUX_PKG_SRCDIR/configure"
```

Comment:
Someone said we do not want to depend on perl in our scripts. Probably it applies here as well.
```suggestion
	sed -i "s/TERMUX_ARCH/${TERMUX_ARCH}/"  "${TERMUX_PKG_SRCDIR}/configure"
```

Reply:
No idea how I've not seen this review before.

<h1><em></em></h1> <!-- thin separator --> 

Yes to this.